testing/iframe.js

```diff
@@ -71,7 +71,7 @@ export function createFixtureIframe(fixture, initialIframeHeight, opt_beforeLoad
     // on that window.
     window.beforeLoad = function(win) {
       // Flag as being a test window.
-      win.AMP_TEST = true;
+      win.AMP_TEST_IFRAME = true;
```

Comment:
Maybe these names should be changed, but I think the semantics are simple:

- `AMP_TEST`: behave specific to a unit test. This should e.g. trigger localhost-y URLs
- `AMP_TEST_IFRAME`: This is a per-test iframe. Behave accordingly.

With that you either always want to set `AMP_TEST` when `AMP_TEST_IFRAME` is set, or make `getMode` look at both variables.